exercises/practice/meetup/meetup.el

```diff
@@ -0,0 +1,14 @@
+;;; meetup.el --- Meetup (exercism)  -*- lexical-binding: t; -*-
+
+;;; Commentary:
+
+;;; Code:
+
+
+(defun meetup (month year dow schedule)
```

Comment:
`dow` is a bit vague. Can we call it `dayofweek` or `weekday`? I'd also like to change the order so it's year month dow schedule. That'd probably be easier for students to parse and it lines up a bit better with the expected format for the return value

Reply:
should be done. Thanks